addons/web/static/src/core/select_menu/select_menu.xml

```diff
@@ -11,14 +11,18 @@
                 onStateChanged.bind="onStateChanged"
                 navigationOptions="{ virtualFocus: this.props.searchable }"
             >
-                <button t-att-class="`o_select_menu_toggler btn btn-light w-100 bg-light ${props.togglerClass || ''} ${canDeselect ? 'o_can_deselect' : ''}`">
+                <button type="button" t-att-class="`o_select_menu_toggler btn btn-light w-100 bg-light ${props.togglerClass || ''} ${canDeselect ? 'o_can_deselect' : ''}`" t-att-disabled="props.disabled">
                     <t t-if="props.multiSelect">
                         <div class="text-wrap text-start">
-                            <TagsList tags="multiSelectChoices"/>
+                            <t t-if="props.placeholder and !props.value.length" t-out="props.placeholder"/>
```

Comment:
I think the placeholder should be displayed differently from a selected value so the user knows no value is currently selected, something like this could be nice:
```js
<span t-if="..." class="text-muted fst-italic" t-out="...">
```

Reply:
@BastienFafchamps This could be nice to have. :smile:  Thanks : ) 
I have adapted the changes.